src/zc/buildout/buildout.py

```diff
@@ -384,7 +384,12 @@ def __init__(self, config_file, cloptions,
 
         download_cache = options.get('download-cache')
         extends_cache = options.get('extends-cache')
+
+        if bool_option(options, 'abi-tag-eggs', 'false'):
+            from zc.buildout.pep425tags import get_abi_tag
+            options['eggs-directory'] += '-' + get_abi_tag()
```

Comment:
BTW, The abi tag captures the Python version, so there aren't abi-dependent eggs. :)

There is a potential savings for platform-independent eggs in cases where there's more than one abi variant for a platform but I don't think that will be the case for most people. (It will be for me, but I can cope.)